app/viewmodels/checkAnswers/sections/items/ItemPackagingProductTypeSummary.scala

```diff
@@ -31,12 +31,12 @@ object ItemPackagingProductTypeSummary {
     request.userAnswers.get(ItemPackagingProductTypePage(itemIdx, packagingIdx)).map {
       answer =>
 
-        val value = if (answer) "site.yes" else "itemPackagingProductType.no"
+        val value = if (answer) "site.yes" else "site.no"
 
         SummaryListRowViewModel(
           key     = "itemPackagingProductType.checkYourAnswersLabel",
           value   = ValueViewModel(value),
-          actions = Seq(
+          actions = if(!ItemsPackagingSectionItems(itemIdx, packagingIdx).isCompleted) Seq() else Seq(
```

Comment:
I think passing an empty Seq through here ends up still keeping the actions column would be but doesn't keeps it empty rather than not having an actions column at all. Not sure if it really matters tho